### PR TITLE
chore: fix docker build

### PR DESCRIPTION
**Pull request by danilaplee**
this fixes the currently failing docker build and adds a check on every pull request that the docker is built properly

### CODE REVIEW

File: packages/evolu-common/package.json
Comment (steida) [inline packages/evolu-common/package.json:1]: Why those two new dependencies are required?

Comment (danilaplee) [reply packages/evolu-common/package.json:1]: Those dependencies are actually peer dependencies of other libraries and are already installed but in node.js they are not hoisted

Comment (steida) [reply packages/evolu-common/package.json:1]: But why are they in `evolu-common`?

Comment (danilaplee) [reply packages/evolu-common/package.json:1]: Because they are dependencies of evolu common as i understand, though we can move them somewhere else

Comment (steida) [reply packages/evolu-common/package.json:1]: The point of `evolu-common` is to contain the absolute minimum of what is needed. Everything else belongs to other packages :-)

Comment (danilaplee) [reply packages/evolu-common/package.json:1]: Well where should this be moved if it is a dependecy of evolu-common?

Comment (steida) [reply packages/evolu-common/package.json:1]: Where evolu-common uses it?

Comment (steida) [reply packages/evolu-common/package.json:1]: I see those two dependencies in the Docker file, but I don't know why they are required. No Evolu package uses them.

Comment (danilaplee) [reply packages/evolu-common/package.json:1]: removed

Comment (danilaplee) [reply packages/evolu-common/package.json:1]: @steida can you check this again and merge?

**Comment by steida** [comment]
@danilaplee Thank you!

**Pull request merged**